### DISCUSSION TITLE
X10 ISRM TWIN Lite Pro RF module mod

### DISCUSSION THREAD

**Discussion by Crunch69** [FrSkyRC/ETHOS-Feedback-Community#3483] [Ideas]
Can we breath new life into the horus by adding a TWIN module.  I am hoping it can be done considering the new radios are toy like and lack the quality the horus radios have.

**Comment by strgaltdel** [comment]
did you check the X20pro ?

another option could be to wait for an x24

personally, i wouldn't opt for such a twin update
on the long / mid run, the horus series with its old cortex series and the resulting ethos restrictions wouldn't fullfillmy needs

i've both X18se and x12s
UI performance , lua runtimes etc.. is much better on the newer hw
